stackstorm-nexus3/README-test.md

```diff
@@ -0,0 +1,97 @@
+## Testing locally
+
+
+Copy component files
+    
+    cp -rf actions/*  /opt/stackstorm/packs/nexus3/actions/
```

Comment:
Does `st2 pack install file://$(pwd)` not work here? I see you use it later on.

Reply:
Thank you so much for taking time to review it.  
This file i've created to test actions using st2-docker. `st2 pack` does work but it require me to add `git commit` first . Also, it takes a lot of time to finish.  Using copy file i find my `.py` files updated instantly without having to do any reload.
I do `reload` only if any `yaml` file is edited. In such cases i used `st2 pack instant` with a dirty `git commit` 

But I"ll re-edit all the readme file once all the tests are written and passed

Reply:
README-test file deleted and kept the only Makefile for the code to do all the talking